screwdriver/screwdriver.go

```diff
@@ -270,6 +271,10 @@ func (a api) put(url *url.URL, bodyType string, payload io.Reader) ([]byte, erro
 	return a.write(url, "PUT", bodyType, payload)
 }
 
+func (a api) GetAPIURL() string {
+	return a.baseURL
```

Comment:
I think the intention of having `template-main` check for the `SD_API_URL` (https://github.com/screwdriver-cd/template-main/blob/46769f6ba2e3956c3d22c07cb5ad9a02d819fda9/index.js#L25) was to make it flexible so if others deploy their own version of Screwdriver, it'll just use their API URL, which I think might not include a version.

Right now, `makeURL` seems to be hardcoding "v4" as the version in the URL.